### DISCUSSION TITLE
SyntaxError: invalid syntax in cycle for

### DISCUSSION THREAD

**Discussion by jasperDD** [apache/incubator-mxnet#14225] [Q&A]
test='C:\Users\Admin\Desktop\test.lst' # folder where must be my test.lst

test='C:/Users/Admin/Desktop/test.lst'
DataSet='C:/Users/Admin/Downloads/mypic/'

data_iter = mx.image.ImageIter(
batch_size=4,
data_shape=(3,816, 1232),
label_width=4,
path_imglist='C:/Users/Admin/Desktop/test.lst',
path_root='DataSet')

for data in data_iter: d = data.data[0] 
break

error
==========================
>>> for data in data_iter: d = data.data[0]
... break
  File "<stdin>", line 2
    break
        ^
SyntaxError: invalid syntax
>>>
=================================

>>> import numpy as nd
>>> img =nd.transpose(d,(0,2,3,1))
Traceback (most recent call last):
Error
====
  File "<stdin>", line 1, in <module>
NameError: name 'd' is not defined
>>> d
How to fix it?

[a comment by an automated account is not shown]

**Comment by frankfliu** [comment]
@mxnet-label-bot add [Question]
